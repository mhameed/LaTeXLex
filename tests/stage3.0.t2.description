author: mh
date: 09/03/23
description: tests preservation of dollar env and reformatting 
everywhere else

