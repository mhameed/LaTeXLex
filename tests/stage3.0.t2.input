This is a line
That spanns over
multiple lines. \bigskip This is a line
That spanns over
multiple lines.

\bigskip \bigskip \bigskip \bigskip

This is a totally new paragraph.
That has some 
$$f(x) := x^2 + x
+ 100x +2
$$ at the end of it.

