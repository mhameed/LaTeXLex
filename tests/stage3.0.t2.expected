This is a line That spanns over multiple lines. 


This is a line That spanns over multiple lines.


This is a totally new paragraph. That has some 
f(x) := x^2 + x
+ 100x +2
 at the end of it.


